sale_order_line_packaging_qty/models/sale_order_line.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_round
+
+
+class SaleOrderLine(models.Model):
+
+    _inherit = "sale.order.line"
+
+    product_packaging_qty = fields.Integer(
```

Comment:
This should be Float

Reply:
In my opinion it shouldn't be float, there was a comment in the previous suggesting to make it integer. As a seller if I configure packaging to sell my product I want to sell it that way, otherwise in my opinion it complicates it much more